packages/twenty-front/src/pages/settings/data-model/SettingsObjects.tsx

```diff
@@ -29,7 +39,25 @@ import { Table } from '@/ui/layout/table/components/Table';
 import { TableHeader } from '@/ui/layout/table/components/TableHeader';
 import { TableSection } from '@/ui/layout/table/components/TableSection';
 import { UndecoratedLink } from '@/ui/navigation/link/components/UndecoratedLink';
-
+import { useSortedArray } from '~/hooks/useSortedArray';
+export enum SortKeys {
```

Comment:
SortKeys should be more self-explanatory, I cannot understand at first glance what it talks about. 

Is it AvailableSortFieldName ?